public/js/src/module/form-cache.js

```diff
@@ -7,7 +7,10 @@ import events from './event';
 import settings from './settings';
 import connection from './connection';
 import assign from 'lodash/assign';
-import encryptor from './encryptor';
```

Comment:
As with many of the changes in `connection.js`, the changes here remove last-saved implementation details, which are now implemented in `last-saved.js`.